### PR TITLE
Add test cases for document and workspace symbols

### PR DESCRIPTION
**Pull request by nadeeshaan**

### CODE REVIEW

File: language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/docsymbol/DocumentSymbolTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * Ballerina Language Server Workspace Test.
+ * 
+ * @since 0.982.0
```
Comment (rasika) [inline language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/docsymbol/DocumentSymbolTest.java]: Shouldn't it be 0.981.2?

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/docsymbol/DocumentSymbolTest.java]: Should be the next major release going to add

File: language-server/modules/langserver-core/src/test/resources/completion/annotation/endpointAnnotation1.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "items": [
+    {
+      "label":"swagger:ClientEndpoint",
+      "detail":"Annotation",
+      "sortText":"110",
+      "insertText":"swagger:ClientEndpoint",
+      "insertTextFormat":"Snippet"
+    }
+  ]
+}
```
Comment (rasika) [inline language-server/modules/langserver-core/src/test/resources/completion/annotation/endpointAnnotation1.json]: Shall we add a line feed here

Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/resources/completion/annotation/endpointAnnotation1.json]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10205

**Pull request merged**